website/docs/api/hierarchy.mdx

```diff
@@ -220,7 +220,118 @@ function example() {
 </TabItem>
 </Tabs>
 
-Returns the center child (finer) index contained by `cell` at resolution `childRes`.
+Provides the center child (finer) index contained by `cell` at resolution `childRes`.
+
+Returns 0 (`E_SUCCESS`) on success.
+
+## cellToChildPos
+
+<Tabs
+  groupId="language"
+  defaultValue="c"
+  values={[
+    {label: 'C', value: 'c'},
+    {label: 'Python', value: 'python'},
+    {label: 'Java', value: 'java'},
+    {label: 'JavaScript (Live)', value: 'javascript'},
+  ]
+}>
+<TabItem value="c">
+
+```c
+H3Index cellToChildPos(H3Index child, int parentRes, int64_t *out);
```

Comment:
All the other bindings say "cell" - maybe they should also have the parameter named "child"?

Reply:
This is all prospective, of course, as the binding owners may have other ideas, but I can update all to `child` for now